index.html

```diff
@@ -290,7 +288,24 @@ <h3 id="technology_specific_solutions">Technology-specific solutions</h3>
 <p> This combines both language and direction metadata and, if consistently adopted, makes interchange between different formats easier. Consistency between different specifications and document formats allows for the easy interchange of string data. By naming field attributes in the same way and adopting the same semantics, different specifications can more easily extract values from or add values into resources from other data sources.</p>
 </section>
 
+<section id="protocol-strings">
+<h3>Strings that are part of a legacy protocol or format</h3>
+
+<p class="advisement">For strings that cannot specify language and direction due to legacy format reasons, specifications SHOULD specify that base direction depends on first-strong heuristics and that language information is undefined (unless specified by some other feature of the specification).</p>
+
+<p>Some string values depend on or are defined by existing protocols or formats that do not provide language or direction metadata for a given string. For example, many HTTP headers define their contents as if they were not <a>localizable text</a>, even when, in some cases, they contain natural language text. When a specification has to take such a dependency or define a format that describes one of these strings, there will be no language or direction metadata to associate with the string in the specification's data structure or document format. And any metadata that the specification's data structure or document format provides will not be serialized through the underlying format.</p>
+
+<aside class="example">
+<p>The following <code>dictionary</code> defined in the [[Webtransport]] specification depends on a data structure defined in [[WEB-TRANSPORT-HTTP3]]. The field <code>reason</code> is simply a UTF-8 encoded byte array. Since the underlying protocol does not provide fields or encoding for language or direction metadata, it is not possible to accurately derive the values for inclusion in the dictionary and any values generated by producers implementing WebTransport would be dropped on the floor when the structure is eventually serialized on the wire.</p>
```

Comment:
`WEB-TRANSPORT-HTTP3` should probably be `RFC9000`?

Reply:
Probably? I followed the links in their docs and found the I-D (rather than an RFC). I'll check it out and make the change if appropriate.

Reply:
Done.